cogs/roles.py

```diff
@@ -1,6 +1,10 @@
 import discord
 from discord.ext import commands
 from utils.checks import cogify, admin_command
+import logging
+
+logger = logging.getLogger('bot')
+logger.setLevel(logging.DEBUG) # TODO: change back to logging.info
```

Comment:
There should be a separate logger for cogs - see [here ](https://github.com/ib-ai/ib.py/blob/4231101f9100172eeb1ba49c2dd861a6046e6424/cogs/monitor.py#L13)for implementation.